### ISSUE TITLE
Audio quality needed to train the model? Using mp3 files?

### ISSUE THREAD

**Issue by Quadraaa** [keithito/tacotron#122]
Hi!

This is a great implementation, thanks a lot!

Could someone please share their experience, which audio quality is needed to train the model? Will it be able to produce speech of at least low/medium quality if trained with 128 kbps .mp3 audio files from an audio book recorded by a professional speaker? The records sound clean to a human ear, there is no background noise.

Will 256 kbps .mp3 be better? Or do I need to use uncompressed .wav only in order to get any feasible results?

**Comment by keithito** [comment]
128 kbps mp3 should be fine -- that's what the LJ speech dataset was derived from. Would uncompressed .wav be even better? Possibly. If you are recording new prompts, I'd go with .wav.

**Comment by Quadraaa** [comment]
@keithito 

Thanks for the answer! 

Trained on my non-English dataset. The model produces speech, but there are a lot of artefacts (the largest ones are that the model repeats words or parts of the phrase at the end or pronounces the phrase in very low volume and after the end of the phrase produces some weired loud noise for several seconds). The same problem with repeating the last part of the sentence occured when I used Nancy dataset for training. Isn't the model penalized heavily for producing the sequence which is longer than the ground truth sequence? Maybe this is a known issue with a known solution?

Can the quality of generated speech decrease with more steps of training? I.e. can the model overfit to the data if the data is, say, 15+ hours? Maybe it is worth creating a validation set and monitoring the validation error, or we can just train and produce eavluation audio files to monitor the quality of generated speech?

**Comment by keithito** [comment]
Yes, please make sure to trim any silences from the ends of your samples. That can help with the alignment. Also, you can try using the location sensitive attention mechanism from here:
https://github.com/keithito/tacotron/blob/tacotron2-work-in-progress/models/attention.py

It's possible to overfit if you don't have enough training data or there's not enough diversity in the training data. A validation set would probably be good.

**Comment by Quadraaa** [comment]
Thanks, will try to cut the silence and use the location sensitive attention mechanism. I've actually cut the silence, but maybe it should be cut more tight.

**Comment by v-yunbin** [comment]
@keithito trimming any silences from the starts of own samples can also help alignment? or the silences from the starts have no effect on the alignment?

**Comment by keithito** [comment]
Yes. Trimming should definitely help.
